app/scripts/controllers/main/MainController.js

```diff
@@ -6,7 +6,41 @@
             $http.get('release.json').success(function(data) {
                 scope.version = data.version;
                 scope.releasedate = data.releasedate;
-            } );
+            });
+
+            scope.islogofoldernamefetched = false;
+            scope.islogofoldernameconfig = false;
+            scope.isFaviconPath = false;
+            scope.isHeaderLogoPath = false;
+            scope.isBigLogoPath = false;
+            scope.isLargeLogoPath = false;
+
+            if(!scope.islogofoldernamefetched && $rootScope.tenantIdentifier && $rootScope.tenantIdentifier != "default"){
```

Comment:
There is no tenant specific filter is added. This config will have more than 1 tenants logos.